### PR TITLE
update what is tizen iot

### PR DESCRIPTION
**Pull request by jay4peace**
### Change Description ###

 - Fix broken links of iot api set
 - Add links for developing applications and createing images sections

### CODE REVIEW

**Comment by safir-srbd** [comment]
@jay4peace Hi. Hope you are well. I closed and reopened the pull request to trigger staging rebuild.

File: docs/platform/what-is-tizen/profiles/iot.md [hunk 1]
```diff
@@ -13,28 +13,30 @@ Tizen IoT supports the following environment for development:
 
 ## Develop applications for Tizen IoT
 To develop applications for the IoT profile:
-- [Create Your First Tizen IoT .NET Application](../../../application/dotnet/get-started/iot/first-app.md)
+- [Create Your First Tizen IoT .NET Application](../../../application/dotnet/get-started/first-app.md)
+- [Create Your First Tizen IoT Native Service Application](../../../application/native/get-started/iot/first-app.md)
 
 ## Create Tizen IoT images
-Tizen IoT provides a building block pool of components mainly based on the [Tizen native API sets](../../../application/native/api/wearable/latest/index.html).
+Tizen IoT provides a building block pool of components mainly based on the [Tizen native API sets](../../../application/native/api/iot-headed/latest/index.html).
```
Comment (arisha-srbd) [inline docs/platform/what-is-tizen/profiles/iot.md:20]: API names should be capitalized. Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/naming-rules.md
```suggestion
Tizen IoT provides a building block pool of components mainly based on the [Tizen Native API sets](../../../application/native/api/iot-headed/latest/index.html).
```

File: docs/platform/what-is-tizen/profiles/iot.md [hunk 2]
```diff
@@ -13,28 +13,30 @@ Tizen IoT supports the following environment for development:
...
 
-**Building block prsets for IoT core:**
+**Building block persets for IoT core:**
```
Comment (arisha-srbd) [inline docs/platform/what-is-tizen/profiles/iot.md:22]: The word “presets” has been misspelled here. 
```suggestion
**Building block presets for IoT core:**
```

File: docs/platform/what-is-tizen/profiles/iot.md [hunk 3]
```diff
@@ -13,28 +13,30 @@ Tizen IoT supports the following environment for development:
...
 - The minimum set of bootable modules that make up the Tizen platform image for IoT devices.
 - The preset contains core components that are required for a basic system image to boot.
 
-**Building block prsets for Headless:**
+**Building block persets for Headless:**
```
Comment (arisha-srbd) [inline docs/platform/what-is-tizen/profiles/iot.md:26]: The word “presets” has been misspelled here. 
```suggestion
**Building block presets for Headless:**
```

File: docs/platform/what-is-tizen/profiles/iot.md [hunk 4]
```diff
@@ -13,28 +13,30 @@ Tizen IoT supports the following environment for development:
...
 - The set of modules that make up the Tizen platform image for headless IoT devices without a display.
-- The preset contains IoT core presets with other components specifically required for headless image.
-- Example of specific components for headless image includes Audio modules, Bluetooth modules, and so on.
+- The preset contains IoT core presets with other components specifically required for headless images.
+- Example of specific components for a headless image includes Audio modules, Bluetooth modules, and so on.
 
-**Building block prsets for Headed:**
+**Building block persets for Headed:**
```
Comment (arisha-srbd) [inline docs/platform/what-is-tizen/profiles/iot.md:31]: The word “presets” has been misspelled here. 
```suggestion
**Building block presets for Headed:**
```

**Review by arisha-srbd — COMMENTED** [review]
@jay4peace hello! Hope you are doing well. I reviewed the updated parts of this PR and made some in-line correction suggestions. Please incorporate the changes. Thank you.

**Review by arisha-srbd — APPROVED** [review]
@jay4peace thank you for incorporating all the changes. It looks good now. 

@asm-foysal @mijin85cho I am approving this PR for merging.

**Pull request merged**